hack/label-custom-kubelet.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -e
+
+# expect oc to be in PATH by default
+OC_TOOL="${OC_TOOL:-oc}"
+
+# Label worker MCP
+echo "[INFO]: Labeling worker MCP with custom-kubelet"
+mcp=$(${OC_TOOL} get mcp --selector='pools.operator.machineconfiguration.openshift.io/worker=' -o name)
+
+${OC_TOOL} label --overwrite "$mcp" custom-kubelet=enabled
```

Comment:
we should minimize node reboots, because we should try hard to make the e2e lane as fast as possible. Fast lane -> we ran it often -> good signal, less bitrot.

Reply:
hh, do you have proposals? Like probably we can do this kind of configuration from test:
1. Pause MCP
2. Create NRO CR
3. Create custom kubeletconfig and label the MCP
4. Unpause the MCP

Reply:
Sorry but why we would have node reboot here? it's just labeling the MCP

Reply:
@cynepco3hahue  yes, pausing, then creating NRO, then create custom kubelet config+ label (or the previous two in any order) then unpause is propably the best we can do. I'd not complicate the flow any further until we try this out.
@Tal-or the labeling per se won't of course cause a reboot, and neither will the kubeletconfig alone, but once MCO notices that both the `kubeletconfig` matches a MCP through the label, it will trigger a reboot.

Reply:
I see. so we can save one reboot by applying this changes and the one from https://github.com/openshift-kni/numaresources-operator/commit/cb2d540cf0a87a63def2fcfcf6373ef5dbfe9258 together.